templates/definition/meter/sonnenbatterie.yaml

```
@@ -26,6 +26,18 @@ params:
       de: API Token (benötigt für aktive Batteriesteuerung)
       en: API Token (required for active battery control)
     usages: ["battery"]
+  - name: backupbuffer
+    required: false
+    help:
+      de: In der Batterie verbleibende Reserve-Kapazität (in %)
+      en: default remaining battery charge (in %)
+    default: 10
```

```suggestion
      de: Verbleibende Reservekapazität in der Batterie (in %)
      en: Remaining reserve capacity in the battery (in %)
    default: 0
    usages: ["battery"]
```
* I suggest using 0 as default, as I assume most users do not own a SonnenProtect backup box. Then a value different from 0 does not make much sense.
* Adding `usages: ["battery"]` is showing this parameter only in the battery documentation.
* Minor rewording of help text